test/cases/provider-id.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "--> Should fetch imds details correctly"
+EXPECTED_INSTANCE_ID="i-1234567890abcdef0"
+EXPECTED_AVAILABILITY_ZONE="us-east-1a"
+EXPECTED_PROVIDER_ID="aws://$EXPECTED_AVAILABILITY_ZONE/$EXPECTED_INSTANCE_ID"
```

Comment:
From what I can tell, our provider ID has 3 slashes. Actual provider ID: `aws:///us-west-2d/i-052b1a849f1d6c78d
`.

Reply:
Ah, yeah, you're right. Will fix.